packages/time-picker/src/time-picker-com/panel-time-range.vue

```diff
@@ -276,6 +276,7 @@ export default defineComponent({
     }
 
     const formatToString = value => {
+      console.log(value, props.format)
```

Comment:
is this necessary?

Reply:
Nope. This was my bad. I will add an plugin to remove `console.log` statement to make sure this won't happen again

Reply:
I just fixed it. Upgrade your local version then it should be resolved.